skipper.go

```diff
@@ -1521,24 +1524,26 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 
 	o.PluginDirs = append(o.PluginDirs, o.PluginDir)
 
-	tracer, err := o.openTracingTracerInstance()
+	tracer, err := o.tracerInstance()
 	if err != nil {
 		return err
 	}
 
-	// tee filters override with initialized tracer
-	o.CustomFilters = append(o.CustomFilters,
-		// tee()
-		teefilters.WithOptions(teefilters.Options{
-			Tracer:   tracer,
-			NoFollow: false,
-		}),
-		// teenf()
-		teefilters.WithOptions(teefilters.Options{
-			NoFollow: true,
-			Tracer:   tracer,
-		}),
-	)
+	// tee filters override if we have a tracer
+	if len(o.OpenTracing) > 0 {
```

Comment:
does the override only apply for opentracing?

Reply:
This option has a very unfortunate name, but it contains both OpenTracing and OpenTelemetry tracers. I could not change the name to something with more meaning than `Options.OpenTracing` to not break backward compatibility with library users, which is not great but we don't have much option. :/